src/main/scala/it/pagopa/interop/apigateway/api/impl/GatewayApiServiceImpl.scala

```diff
@@ -241,26 +199,22 @@ final case class GatewayApiServiceImpl(
     toEntityMarshallerEServices: ToEntityMarshaller[EServices],
     toEntityMarshallerProblem: ToEntityMarshaller[Problem]
   ): Route = {
+    val operationLabel =
+      s"Retrieving Organization EServices for origin $origin externalId $externalId attributeOrigin $attributeOrigin attributeCode $attributeCode"
+    logger.info(operationLabel)
+
     val result: Future[EServices] = for {
-      tenant       <- tenantManagementService.getTenantByExternalId(origin, externalId)
-      attribute    <- attributeRegistryManagementService.getAttributeByOriginAndCode(attributeOrigin, attributeCode)
-      eServices    <- catalogManagementService.getEServices(tenant.id, attribute.id)
-      apiEServices <- Future.traverse(eServices)(enhanceEService)
+      tenant    <- tenantManagementService.getTenantByExternalId(origin, externalId)
+      attribute <- attributeRegistryManagementService.getAttributeByOriginAndCode(attributeOrigin, attributeCode)
+      eServices <- catalogManagementService.getEServices(tenant.id, attribute.id)
+      allowedEServices = eServices.filter(_.descriptors.nonEmpty)
```

Comment:
Bug

Use case:
- an organization performs a legit retrieve of another organization eservices
- one or more returned eservices do not contain descriptors
- the request fails because [here](https://github.com/pagopa/interop-be-api-gateway/pull/133/files?diff=split&w=1#diff-45c4cb111d08dd8f28095838b0a597b24d6249d2241581517ee286af10975124R170) an error is thrown